src/screens/TokenDetail.js

```diff
@@ -52,29 +56,67 @@ class TokenDetail extends React.Component {
     walletAmount: 0,
     action: '',
     successMessage: '',
+    totalSupply: 0,
+    canMint: null,
+    canMelt: null,
+    paramUID: null,
+    unknown: null,
```

Comment:
Naming a variable `unknown` seems like we're trolling future developers who'll have to maintain this code 🤣 

I think `isUnknown` would be a better name.

Reply:
I don't really think it's trolling because we have the description of each state, so people can understand it. But I agree that the name could be better, so I think I will change to `tokenUnknown`. What do you think?

I feel like `unknown` and `isUnknown` is pretty much the same

Reply:
`tokenUnknown` sounds good.